docs/hmi_changelog.md

```diff
@@ -4,6 +4,14 @@
 
 - PLC properties can only be used explicitly in the HMI by linking the property individually to a control attribute. If the entire function block is linked to a control attribute, the property is not called. This is the case when a function block is used as the source data of the DataGrid or as a user control parameter.
 
+## Version 1.12.756.1
+
+- EtherCAT Diagnostics has now graphs for processdata, similar to the online view in the IO tree in a TwinCat project.
+- Scope control has new functionality to download/upload recordings.
+- Added functions
```

Comment:
What do you mean here? Are two functions `Locale` and `GetLocalizedText` added? If yes, can you format them as follows:

> `FunctionName`: explanation what it does.

Reply:
"Locale" is the function category. I changed it to a more clear format.